internal/gamepaddb/gamepaddb.go

```diff
@@ -519,11 +519,12 @@ func buttonValue(id string, button StandardButton, state GamepadState) float64 {
 	return 0
 }
 
-func IsButtonPressed(id string, button StandardButton, state GamepadState) bool {
-	// Use XInput's trigger dead zone.
-	// See https://source.chromium.org/chromium/chromium/src/+/main:device/gamepad/public/cpp/gamepad.h;l=22-23;drc=6997f8a177359bb99598988ed5e900841984d242
-	const threshold = 30.0 / 255.0
+// Use XInput's trigger dead zone.
```

Comment:
Start the comment with `ButtonPressedThreshold represents ...` for the Go convention.